xml/bp_troubleshooting_nfs.xml

```diff
@@ -13,7 +13,131 @@
+  Sep 22 04:59:08 node1 bash[29705]: 22/09/2020 02:59:08 : epoch 5f6960b1 : node1 : ganesha.nfsd-1[svc_6] export_check_access :EXPORT :M_DBG :EXPORT_DEFAULTS (options=03303002/00080000 , , , > ​
+  Sep 22 04:59:08 node1 bash[29705]: 22/09/2020 02:59:08 : epoch 5f6960b1 : node1 : ganesha.nfsd-1[svc_6] export_check_access :EXPORT :M_DBG :default options (options=03303002/ffffffff root_squash , ----, 34-, UDP, TCP,> ​
+  Sep 22 04:59:08 node1 bash[29705]: 22/09/2020 02:59:08 : epoch 5f6960b1 : node1 : ganesha.nfsd-1[svc_6] export_check_access :EXPORT :M_DBG :Final options (options=03303002/ffffffff root_squash , ----, 34-, UDP, TCP,> ​
+  Sep 22 04:59:08 node1 bash[29705]: 22/09/2020 02:59:08 : epoch 5f6960b1 : node1 : ganesha.nfsd-1[svc_6] export_check_access :RW LOCK :F_DBG :Unlocked 0x7f0872a64660 (&amp;export_opt_lock) at /home/abuild/rpmbuild/BUILD/nfs-> ​
+  Sep 22 04:59:08 node1 bash[29705]: 22/09/2020 02:59:08 : epoch 5f6960b1 : node1 : ganesha.nfsd-1[svc_6] get_gsh_client :RW LOCK :F_DBG :Got read lock on 0x7f0872a5e5d0 (&amp;client_by_ip.lock) at /home/abuild/rpmbuild/BUILD> ​
+  Sep 22 04:59:08 node1 bash[29705]: 22/09/2020 02:59:08 : epoch 5f6960b1 : node1 : ganesha.nfsd-1[svc_6] get_gsh_client :HT CACHE :DEBUG :client_mgr cache hit slot 859
+</screen>
+   <sect2 xml:id="troubleshooting-set-log-level">
+     <title>Setting the Log Level</title>
+     <para>
+       To change the default debug level, you can use a &rados; configuration
+       object and configure a &ganesha; parameter.
+     </para>
+     <para>
+       To debug logging, change the log level to <literal>FULL_DEBUG</literal>
+       to increase the log verbosity. Keep in mind that this can produce
+       large amounts of data in journalctl and adversely affect performance.
+     </para>
+     <para>
+       Format a log block, for example:
+     </para>
+<screen>
+  LOG { ​
+  # The components block contains one or more logging components ​
+  # and the setting to be used. ​
+  components { ​
+  # The ALL component is special. When set it defines the level ​
+  # for all components and overrides any other setting in this block. ​
+  ALL = FULL_DEBUG; # this will likely kill performance ​
+  } ​
+</screen>
+     <para>
+       Put the file into &rados;. In this example, the file is named
+       <filename>debugconf-nfs.foo</filename>:
+     </para>
+<screen>
+%url "rados://nfs-ganesha/foo/export-1" ​
+%url "rados://nfs-ganesha/foo/userconf-nfs-foo" ​
+%url "rados://nfs-ganesha/foo/debugconf-nfs.foo"​
+</screen>
+     <para>
+       Add a reference to the parameter in your &rados; common configuration file:
+     </para>
+<screen>&prompt.cephuser;rados --pool nfs-ganesha --namespace foo put debugconf-nfs.foo debugconf-nfs.foo​
+&prompt.cephuser;rados --pool nfs-ganesha --namespace foo put conf-nfs.foo conf-nfs.foo​
+</screen>
```

Comment:
add a paragraph here similar to the debugconf RADOS object to put the common config changes into a RADOS object...

e.g. `Put the updated common configuration file to the &rados; common configuration object....`